### PR TITLE
fix: added function to show no bank when the bank option is not available

### LINKED ISSUE #490: Empty view is missing in Link Your Bank Account
**Summary:**

There should be an empty view when no bank is found in Link Your Bank Account.

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes #490

- [.] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [.] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [.] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
@luckyman20 Please review

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/LinkBankAccountActivity.java [hunk 1]
```diff
@@ -156,6 +160,13 @@ private void filter(String text) {
             }
         }
         mOtherBankAdapter.filterList(filteredList);
+        if (filteredList.isEmpty()) {
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/LinkBankAccountActivity.java]: Are you sure it brings empty list only?

Comment (Bhavnaharitsa) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/bank/ui/LinkBankAccountActivity.java]: Yes, it does!

**Comment by Bhavnaharitsa** [comment]
ezgif com-video-to-gif (1)

@luckyman20 Done!

**Comment by Bhavnaharitsa** [comment]
@luckyman20 Please review

**Comment by Bhavnaharitsa** [comment]
@luckyman20 Please review this too

**Comment by naman14** [comment]
Looks good

**Pull request merged**